### PR TITLE
Update workshop list, add pointer to omero-guides

### PR DESCRIPTION
**Pull request by pwalczysko**
Adding the past workshops.

cc @jburel

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -5,6 +5,28 @@ Previous workshops are listed below, with links to training materials for each.
 
  * Also see the main OME website [training pages](https://www.openmicroscopy.org/training/).
  * For details on the training repositories and data download, see [about.md](about.md).
+ * For workshop contents presented in user-friendly manner, see [omero-guides](https://omero-guides.readthedocs.io/en/latest/index.html)
```
Comment (mtbc) [inline README.md]:
```suggestion
 * For workshop contents presented in a user-friendly manner, see [omero-guides](https://omero-guides.readthedocs.io/en/latest/index.html)
```

File: README.md [hunk 2]
```diff
@@ -5,6 +5,28 @@ Previous workshops are listed below, with links to training materials for each.
...
+--------------------------------
+A workshop on OMERO including Fiji, OMERO.iviewer, OMERO.parade and OMERO.figure.
+ * A **1-day** workshop
+ * Workshop intro [slides](https://downloads.openmicroscopy.org/presentations/2019/New-Zealand/OMERO-workshop/)
+ * Workshop: [walkthrough Day 1 pdf](https://downloads.openmicroscopy.org/presentations/2019/New-Zealand/Hamilton_2019_walkthrough.pdf)
+
+2019-10-9-10 Freising, Germany
+------------------------------
+A workshop on OMERO including Fiji, OMERO.iviewer, OMERO.parade and OMERO.figure.
+ * A **2-days** workshop
```
Comment (mtbc) [inline README.md]:
```suggestion
 * A **2-day** workshop
```
also applies below

File: README.md [hunk 3]
```diff
@@ -5,6 +5,28 @@ Previous workshops are listed below, with links to training materials for each.
...
  * Also see the main OME website [training pages](https://www.openmicroscopy.org/training/).
  * For details on the training repositories and data download, see [about.md](about.md).
+ * For workshop contents presented in user-friendly manner, see [omero-guides](https://omero-guides.readthedocs.io/en/latest/index.html)
+
+2019-11-11 Hamilton, New Zealand
+--------------------------------
+A workshop on OMERO including Fiji, OMERO.iviewer, OMERO.parade and OMERO.figure.
+ * A **1-day** workshop
+ * Workshop intro [slides](https://downloads.openmicroscopy.org/presentations/2019/New-Zealand/OMERO-workshop/)
+ * Workshop: [walkthrough Day 1 pdf](https://downloads.openmicroscopy.org/presentations/2019/New-Zealand/Hamilton_2019_walkthrough.pdf)
```
Comment (mtbc) [inline README.md]:
```suggestion
 * Workshop: [walkthrough pdf](https://downloads.openmicroscopy.org/presentations/2019/New-Zealand/Hamilton_2019_walkthrough.pdf)
```

**Comment by jburel** [comment]
Thanks

**Comment by joshmoore** [comment]
Generally, :+1: but can I suggest we link to https://www.openmicroscopy.org/events/ and only update the session content in one place?

**Comment by jburel** [comment]
I think that list now that we have the events up to date is not relevant
I was thinking to look at that in the new year when we go back to work on training materials again

**Pull request merged**